### PR TITLE
rpi-config: Allow setting hdmi_cvt

### PR DESCRIPTION
**Pull request by otavio**
**- What I did**

Allow setting hdmi_cvt from rpi-config

### CODE REVIEW

File: recipes-bsp/bootfiles/rpi-config_git.bb [hunk 1]
```diff
@@ -105,6 +105,9 @@ do_deploy() {
     if [ -n "${HDMI_MODE}" ]; then
         sed -i '/#hdmi_mode=/ c\hdmi_mode=${HDMI_MODE}' $CONFIG
     fi
+    if [ -n "${HDMI_CVT}" ]; then
```
Comment (agherzan) [inline recipes-bsp/bootfiles/rpi-config_git.bb:108]: Could you also add a documentation commit for this? Thanks, Otavio.

Comment (otavio) [reply recipes-bsp/bootfiles/rpi-config_git.bb:108]: Where?

Comment (agherzan) [reply recipes-bsp/bootfiles/rpi-config_git.bb:108]: https://github.com/agherzan/meta-raspberrypi/blob/master/docs/extra-build-config.md would be great.

Comment (otavio) [reply recipes-bsp/bootfiles/rpi-config_git.bb:108]: done.

Comment (agherzan) [reply recipes-bsp/bootfiles/rpi-config_git.bb:108]: That will do. Thanks.

**Review by agherzan — APPROVED** [review]

**Pull request merged**